.github/workflows/ci.yml

```diff
@@ -94,5 +95,37 @@ jobs:
         ls prebuilds/
       name: List prebuilds
 
-    # only if tag found for release
-    #- if defined APPVEYOR_REPO_TAG_NAME (npm run upload)
+    - name: Upload prebuilds to GitHub
+      run: npm run upload
+      if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+      env:
+        GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  npm-publish:
+    needs: build
+    name: Publish to NPM
+    runs-on: ubuntu-16.04
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 15
+      uses: actions/setup-node@v1
+      with:
+        node-version: 15.x
+        registry-url: 'https://registry.npmjs.org'
+    - run: |
+        sudo apt-get install xvfb \
+                             gnome-keyring \
+                             libsecret-1-dev \
+                             dbus-x11 \
+                             python-gnomekeyring
+      name: Install additional dependencies
+
+    - run: npm install
+      name: Setup environment
+
+    - name: Upload to NPM
+      run: npm publish --access public
+      env:
+        NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
I don't currently have access to add additional settings for this repository, so I can't merge this as-is

Reply:
Who can set it or grant you access?